docs/source/starter/lightning_lite.rst

```diff
@@ -0,0 +1,721 @@
+
+            dataloader = DataLoader(MyDataset(...), ...)
+            dataloader = self.setup_dataloaders(dataloader)
+
+            model.train()
+            for epoch in range(args.num_epochs):
+                for batch in dataloader:
+                    optimizer.zero_grad()
+                    loss = model(batch)
+                    self.backward(loss)
+                    optimizer.step()
+
+
+    Lite(...).run(args)
+
+
+That's all. You can now train on any kind of device and scale your training.
+The :class:`~pytorch_lightning.lite.LightningLite` takes care of device management, so you don't have to.
+You should remove any device specific logic within your code.
+Here is how to train on 8 GPUs with `torch.bfloat16 <https://pytorch.org/docs/1.10.0/generated/torch.Tensor.bfloat16.html>`_ precision:
+
+.. code-block:: python
+
+    Lite(strategy="ddp", devices=8, accelerator="gpu", precision="bf16").run(10)
+
+Here is how to use `DeepSpeed Zero3 <https://www.deepspeed.ai/news/2021/03/07/zero3-offload.html>`_ with 8 GPUs and precision 16:
+
+.. code-block:: python
+
+    Lite(strategy="deepspeed", devices=8, accelerator="gpu", precision=16).run(10)
+
+Lightning can also figure it out automatically for you!
+
+.. code-block:: python
+
+    Lite(devices="auto", accelerator="auto", precision=16).run(10)
+
+
+You can also easily use distributed collectives if required.
+Here is an example while running on 256 GPUs.
+
+.. code-block:: python
+
+    class Lite(LightningLite):
+        def run(self):
+
+            # Transfer and concatenate tensors across processes
+            self.all_gather(...)
+
+            # Transfer an object from one process to all the others
+            self.broadcast(..., src=...)
+
+            # The total number of processes running across all devices and nodes.
+            self.world_size
+
+            # The global index of the current process across all devices and nodes.
+            self.global_rank
+
+            # The index of the current process among the processes running on the local node.
+            self.local_rank
+
+            # The index of the current node.
+            self.node_rank
+
+            # Wether this global rank is rank zero.
+            if self.is_global_zero:
+                # do something on rank 0
+                ...
+
+            # Wait for all processes to enter this call.
+            self.barrier()
+
+
+    Lite(strategy="ddp", gpus=8, num_nodes=32, accelerator="gpu").run()
+
+
+.. note:: We recommend instantiating the models within the :meth:`~pytorch_lightning.lite.LightningLite.run` method as large models would cause an out-of-memory error otherwise.
+
+.. note::
+
+    If you have hundreds or thousands of line within your :meth:`~pytorch_lightning.lite.LightningLite.run` function
+    and you are feeling weird about it, this is exactly the expected feeling.
+    This is how our :class:`~pytorch_lightning.core.lightning.LightningModule` started 2-3 years ago
```

Comment:
maybe mention the year since `2-3 years` is relative.
```suggestion
    This is how our :class:`~pytorch_lightning.core.lightning.LightningModule` started in 2019
```

Reply:
Done !